提交第5课作业

jimluo opened the pull request:
存储交易 cid:
bafyreigwk4welb5dnwk4gmprkrm4wpq543wrdksdo6ovilqiyeojrbhe34

轻钱包 cid: 
bafy2bzacea3mypsc5c6u532ntdw2eqhtfwwkn7m3whxfh3mjeqbav3uo3mtim

jimluo committed:
#Feat, 提交1-5课作业

longfeiWan9 committed:
Rename 4.Filecoin&IPFS开发工具支持/Homeworker 4 Jasonwu123.md to 4.Filecoin&IPFS开发工具支持/Homeworker 4/Jasonwu123.md

longfeiWan9 committed:
Delete Jasonwu123.md

longfeiWan9 reviewed (changes requested):
你好像不小心把第六节课的介绍和作业删除了，可以麻烦把他们加回去吗？

jimluo committed:
Merge branch 'ipfs-and-filecoin:main' into main

jimluo committed:
6th NFT homework

Transaction Hash: 0xdab8a9f8496e7b27d5e2230b92252994099f7ff8bb7c4c90f156990060c62832 
Status: Success
Block: 26783466 25 Block Confirmations
Timestamp: 4 mins ago (Jun-17-2022 08:39:59 AM +UTC)
From: 0x1df06b62c1002802e57fed3aaf21281f651d4cee 
To: [Contract 0xb12535e77137380e0d57c7f25375a18ecd1e7ffaCreated] 
Value: 0 MATIC ($0.00)
Transaction Fee: 0.006660595141204614 MATIC ($0.00)
Txn Type: 2 (EIP-1559)

jimluo committed:
Delete 6.通过NFT.storage存储你的NFT/jimluo directory

jimluo committed:
6th

jimluo committed:
Create jimluo

jimluo committed:
Delete jimluo
